interactive.js

```diff
@@ -99,22 +90,24 @@ const preferHeurisicallyInterestingProcesses = (a, b) => {
 };
 
 const filterProcesses = (input, processes, flags) => {
-	const filters = {
-		name: process_ => input ? nameFilter(input, process_) : true,
-		verbose: process_ => input ? (isWindows ? process_.name : process_.cmd).toLowerCase().includes(input.toLowerCase()) : true,
-	};
-
 	const memoryThreshold = flags.verbose ? 0 : 1;
 	const cpuThreshold = flags.verbose ? 0 : 3;
 
-	return processes
+	const filteredProcesses = new FuzzySearch(
+		processes,
+		flags.verbose && !isWindows ? ['cmd'] : ['name'],
```

Comment:
Can be further simplified:

```suggestion
		[flags.verbose && !isWindows ? 'cmd' : 'name'],
```